packages/react-sdk/index.ts

```diff
@@ -1,7 +1,17 @@
+import { setSdkInfo, SfuModels } from '@stream-io/video-client';
+
 export * from '@stream-io/video-client';
 export * from '@stream-io/video-react-bindings';
 
 export * from './src/components';
 export * from './src/contexts';
 export * from './src/hooks';
 export * from './src/types';
+
+// TODO: set valid version
+setSdkInfo({
+  type: SfuModels.SdkType.REACT,
+  major: '0',
+  minor: '0',
+  patch: '0',
+});
```

Comment:
Any way this can be automated? It is too easy to forget bumping up the version while cutting the release. Like - based on tag or branch name etc?

Reply:
Yes, it'll be, we just haven't yet set it up